src/otx/algorithms/visual_prompting/configs/sam_vit_b/config.yaml

```diff
@@ -17,7 +26,12 @@ model:
   freeze_image_encoder: true
   freeze_prompt_encoder: true
   freeze_mask_decoder: false
-  checkpoint: https://dl.fbaipublicfiles.com/segment_anything/sam_vit_b_01ec64.pth
+  checkpoint: pretrained
```

Comment:
It's just a question. if it is set as `pretrained` then proper weight is loaded automatically?